databricks/koalas/frame.py

```diff
@@ -6448,6 +6448,53 @@ def swaplevel(self, i=-2, j=-1, axis=0) -> "DataFrame":
 
         return DataFrame(internal)
 
+    def swapaxes(
+        self, i: Union[str, int] = 0, j: Union[str, int] = 1, copy: bool = True
+    ) -> "DataFrame":
+        """
+        Interchange axes and swap values axes appropriately.
```

Comment:
As this is almost the same as `transpose`, shall we add a note as well? See https://github.com/databricks/koalas/blob/53812b8a2b7ffeaaaed66ea11ccf92d09a6cee05/databricks/koalas/frame.py#L1978-L1991

Reply:
Sounds good! Added.